README.md

```diff
@@ -124,35 +124,37 @@ After installation the reference to these resources has to be added through menu
 
 This is not complicated, but you'll need to work precise. The configuration of HA is stored in .yaml files, these can be edited in the HA file editor (in the left main menu).
 
-  > If you have installed V2G Liberty before, please remove any changes made during that installation to the .yaml files.
+  > For version 0.2.0 the folder structure has changed significantly due to AppDaemon requirements, please check carefully.
 
 After completion of this step you'll end up with a these files and folders (others might be there but are not shown here). Some might be already present and only need editing. Others files or folders might need to added. The files you'll have to edit are marked with *.
 
 ```
 . (root = HA config folder)
-├── appdaemon
-│   ├── apps
-│   │   ├── v2g-liberty
-│   │   │   ├── app_config
-│   │   │   │   ├── table_style.yaml
-│   │   │   │   ├── v2g-liberty-dashboard.yaml
-│   │   │   │   ├── v2g-liberty-package.yaml
-│   │   │   │   ├── v2g_liberty_ui_module_stats.yaml
-│   │   │   │   └── wallbox_modbus_registers.yaml
-│   │   │   ├── constants.py
-│   │   │   ├── flexmeasures_client.py
-│   │   │   ├── get_fm_data.py
-│   │   │   ├── LICENSE
-│   │   │   ├── README.md
-│   │   │   ├── set_fm_data.py
-│   │   │   ├── v2g_globals.py
-│   │   │   ├── v2g_liberty.py
-│   │   │   └── wallbox_client.py
-│   │   └ apps.yaml *
-│   ├── logs (possibly this need to be created)
-│   └ appdaemon.yaml *
+├── packages (possibly this need to be created)
+│   └── v2g-liberty
+│       ├── table_style.yaml
+│       ├── v2g-liberty-dashboard.yaml
+│       ├── v2g-liberty-package.yaml
+│       └── v2g_liberty_ui_module_stats.yaml
 ├── configuration.yaml *
 └── secrets.yaml *
+
+. (root = AppDaemon addon_configs folder, usually called a0d7b954_appdaemon)
+├── apps
+│   ├── v2g-liberty
+│   │   ├── constants.py
+│   │   ├── flexmeasures_client.py
+│   │   ├── get_fm_data.py
+│   │   ├── LICENSE
+│   │   ├── README.md
+│   │   ├── set_fm_data.py
+│   │   ├── v2g_globals.py
+│   │   ├── v2g_liberty.py
+│   │   ├── wallbox_client.py
+│   │   └── wallbox_modbus_registers.yaml
```

Comment:
I would have expected this `yaml` to live next to the other yamls, as was the case before. Then the distinction would also be clearer: yamls go in the `packages` folder, Python files go in the `apps` folder.

Reply:
Then all info would have to go into the secrets.yaml. The config folder cannot be reached from the appdeamon_config folder and visce-versa. And, to be honest, it's contents could be constants in the .py file as it will not change ever..